lib/icedfrisby.js

```diff
@@ -159,7 +159,7 @@ class Frisby {
       after: [],
       finally: [],
       retry: clone.retry || 0,
-      retry_backoff: clone.retry_backoff || 1000,
+      retryBackoff: clone.retryBackoff || clone.retry_backoff || 1000,
```

Comment:
Change to camel-case for consistency. I'm not sure when `retry_backoff` last worked or if it ever worked, so this alias may be unnecessary. Doesn't matter to me either way.